### PR TITLE
Move OS Defines into Config.h

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

**Comment by redhatrises** [comment]
If this change is desired, I can make the changes to the rest of the code base.

**Comment by redhatrises** [comment]
@openscap-ci test this please

File: compat/oscap_platforms.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ * Copyright 2018 OpenSCAP Contributors. See COPYING for License.
```
Comment (jan-cerny) [inline compat/oscap_platforms.h]: There must a full license. Please see other header files.

Comment (redhatrises) [reply compat/oscap_platforms.h]: @jan-cerny done

**Comment by mpreisler** [comment]
I like this change a lot, thx for making the codebase better!

**Pull request merged**